0004/README.md

```diff
@@ -0,0 +1,416 @@
+
+#### Tier one: Self-attestation
+
+The self-attestation tier requires that a build platform:
+
+-  Publish answers to the questionnaire on a domain registered to their
+    organization. They may additionally publish further proof of SLSA
+    conformance (e.g. architecture documentation). This information must be
+    updated at least once every twelve months and any time there is an
+    architectural change to the build platform.
+-  Publish any materials needed to verify provenance signed by the build
+    system on a domain registered to their organization.
+-  Publish vulnerability disclosures for all security incidents that
+    potentially impact the integrity of artifacts or provenance generated by
+    the build system.
+-  Sign and follow the SLSA Conformance Program terms.
+
+The OpenSSF operates this tier through a community-maintained repository of
+build platforms. The maintainers of this repository will be drawn from members
+of the SLSA community who are spread across the industry.  
+In order to participate in this tier, build platforms should 
+
+1. Learn about the SLSA Specification and its requirements.
+1. Create a PR for the repository containing
+    1. Name of the build platform
+    1. Build ID
+    1. Desired SLSA levels
+    1. List of URIs that resolve to proof of SLSA conformance
+    1. List of URIs that resolve to materials for verifying provenance
+    1. List of URIs that resolve to the build platform's public
+        vulnerability disclosures
+
+1. Complete the terms and conditions form.
+1. Send the PR for review.
+
+The community will review the PR and merge it when a quorum of maintainers
+approve it. When reviewing the PR, the community will assume that the
+information in the proof of conformance is truthful, and they will approve it if
+the described build platform meets the requirements for the desired SLSA levels.
+
+Once the PR is merged and the terms are signed, the OpenSSF grants the build
+platform a limited license to use the SLSA logo on its website and other
+promotional materials. Should the build platform fail to adhere to the program's
+terms or meet its requirements, the OpenSSF will revoke that license and, if
+necessary, enforce its trademarks. 
+
+#### Tier two: Third-party audit
+
+The third-party audit tier requires that a build platform:
+
+-  Provide the auditor access to the build system and any supporting
+    materials needed for the auditor to gain trust in the build system.
+-  Accept that the audit report will be made public regardless of the
+    audit's outcome. Audit reports may be delayed up to 90 days to respond to
+    vulnerabilities in the build platform, but they must be published within 30
+    days of patching the vulnerability.
+-  Provide verifiers with any materials needed to verify provenance signed
+    by the build system. Publishing these materials on a domain registered to
+    the build system meets this requirement.
```

Comment:
The idea is to share verification keys along with proof of conformance. Doing so lowers the barrier to verification since you can query the registry to determine whether you trust a build platform and get its public key.

I must admit that I've also been thinking about how slsa-verifier can get additional trust roots without having to hard code them, so I may be conflating the two issues in my head.